django-backend/fecfiler/transactions/fixtures/payload_transaction_pair.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
I'm a little hesitant to put a data file that's not an actual django fixture in the fixures folder.  I would put it in `tests/data/...` but i think that's just a personal preference